ChatSecure/Classes/View Controllers/OTRRoomOccupantsViewController.swift

```diff
@@ -461,7 +461,9 @@ open class OTRRoomOccupantsViewController: UIViewController {
         if let vc = storyboard.instantiateInitialViewController() as? OTRComposeGroupViewController {
             vc.delegate = self
             vc.setExistingRoomOccupants(viewHandler: self.viewHandler, room: self.room)
-            self.navigationController?.pushViewController(vc, animated: true)
+            DispatchQueue.main.async {
+                self.navigationController?.pushViewController(vc, animated: true)
+            }
```

Comment:
If this is happening off of the main thread, all of the other UIKit code above should be put on the main thread as well. Proper fix will be to ensure that addMoreFriends isn't being called off the main queue from the call site. However, it looks like this code is only ever triggered from `didSelectHeaderCell` which is only called from `    public func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath)` which is guaranteed to be on the main thread.

Under what conditions is `addMoreFriends` called off the main thread?